packages/web_ui/src/components/InstanceStatusTag.jsx

```diff
@@ -0,0 +1,21 @@
+import { Tag } from "antd";
+import React from "react";
+
+const statusColors = {
+	unknown: "#d46b08",
+	unassigned: "#d4380d",
+	stopped: "#cf1322",
+	starting: "#7cb305",
```

Comment:
After running this on my dev cluster for a few days I have found the colors for starting and running to be a bit hard to tell appart. ![colors](https://i.imgur.com/EL40aq5.png)

Reply:
Maybe skew the "starting" closer to yellow?

Reply:
Changed the colors to these:
![image](https://user-images.githubusercontent.com/17990055/142064365-83fb83a4-0c40-4c2e-a81d-29fce37454f8.png)